pkg/cmd/release/download/download.go

```diff
@@ -241,6 +246,23 @@ func downloadAsset(httpClient *http.Client, assetURL, destinationPath string, ar
 		return api.HandleHTTPError(resp)
 	}
 
+	var destinationPath = filepath.Join(destinationDir, fileName)
+
+	if len(fileName) == 0 {
+		contentDisposition := resp.Header.Get("Content-Disposition")
+
+		_, params, err := mime.ParseMediaType(contentDisposition)
+		if err != nil {
+			return err
```

Comment:
@mislav Just returning the error here. WDYT of using the tag name in case the content-disposition header is missing?

Reply:
No, I think it's fine to error out. Maybe wrap the error like:
```suggestion
			return fmt.Errorf("unable to parse file name of archive: %w", err)
```